OpenRA.WindowsLauncher/OpenRA.WindowsLauncher.csproj

```diff
@@ -1,35 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>winexe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
-    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <ApplicationIcon>$(LauncherIcon)</ApplicationIcon>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Optimize>true</Optimize>
-    <UseVSHostingProcess>false</UseVSHostingProcess>
-    <LangVersion>7.3</LangVersion>
-    <DebugSymbols>true</DebugSymbols>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
-    <OutputPath>../bin</OutputPath>
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <ExternalConsole>false</ExternalConsole>
-    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <CodeAnalysisRuleSet>..\OpenRA.ruleset</CodeAnalysisRuleSet>
-    <Configurations>Release;Debug</Configurations>
     <AssemblyName>$(LauncherName)</AssemblyName>
   </PropertyGroup>
-  <ItemGroup>
-    <!-- Work around an issue where Rider does not detect files in the project root using the default glob -->
-    <Compile Include="**/*.cs" Exclude="$(DefaultItemExcludes)" />
-  </ItemGroup>
-  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <Optimize>false</Optimize>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetPlatform)' == 'win-x86'">
-    <Prefer32bit>true</Prefer32bit>
-  </PropertyGroup>
+
   <ItemGroup>
     <AdditionalFiles Include="../stylecop.json" />
```

Comment:
Can be removed?

Reply:
Yes, we can remove Prefer32bit. Netcore ignoring this property [dotnet#5074](https://github.com/dotnet/project-system/issues/5074)

Reply:
I mean the line `<AdditionalFiles Include="../stylecop.json" />` which is already defined in the `Directory.Build.props `